contracts/modules/TransferManager/VolumeRestrictionTM.sol

```diff
@@ -449,12 +461,28 @@ contract VolumeRestrictionTM is VolumeRestrictionTMStorage, ITransferManager {
         require(_holder != address(0), "Invalid address");
         require(individualRestriction[_holder].endTime != 0, "Not present");
         individualRestriction[_holder] = VolumeRestriction(0, 0, 0, 0, RestrictionType(0));
+        _deleteHolderFromList(_holder, uint8(TypeOfPeriod.OneDay));
         userToBucket[_holder].lastTradedDayTime = 0;
         userToBucket[_holder].sumOfLastPeriod = 0;
         userToBucket[_holder].daysCovered = 0;
         emit IndividualRestrictionRemoved(_holder);
     }
 
+    function _deleteHolderFromList(address _holder, uint8 _typeOfPeriod) internal {
```

Comment:
Add comment code. specifically, mention that `_typeOfPeriod` is what should NOT be deleted.